packages/expo-video/src/VideoView.web.tsx

```diff
@@ -2,20 +2,31 @@ import React, { useEffect, useRef, forwardRef, useImperativeHandle } from 'react
 import { StyleSheet } from 'react-native';
 
 import { VideoPlayer, VideoViewProps } from './VideoView.types';
+
+/**
+ * This audio context is used to mute all but one video when there is multiple video views playing from one player simultaneously.
+ * Using audio context nodes allows us to mute the videos without showing the mute icon in the video player.
+ */
+const audioContext = new window.AudioContext();
```

Comment:
Calling browser-specific things like `window`, `document` or `navigator` in the top level scope is dangerous as it can easily break server side rendering. We need to check if `typeof window !== 'undefined'`.